{% extends "base.html" %}

{% block title %}About{% endblock %}

{% block app_content %}
    <div>
        <div id="content-padding" class="pl-4 pt-4">
            <h1 id="about-geneplexus">About GenePlexus <sup>&beta;eta</sup></h1>
            <h2 id="description">Description</h2>
            <p>The GenePlexus webserver enables users to predict novel genes similar to their genes of interest based on their patterns of connectivity in human genome-scale molecular interaction networks.</p>
            <p>A user can supply a list of genes of their interest to GenePlexus and select a gene network of their choice. GenePlexus will then train a machine learning model that captures the patterns of network connectivity of the user-defined genes in contrast to other genes in the network. This machine learning model is then used to return a prediction of how associated every gene in the network is to the input gene list based on their network connectivity patterns. Additionally, GenePlexus also enables the user to interpret the custom trained machine-learning model by comparing it to pretrained models for various bioloical processes (defined based on the <a href="http://geneontology.org/">Gene Ontology</a>) and diseases (defined based on curations from <a href="https://www.disgenet.org/">DisGeNet</a> terms). Users can visualize the top predictions in the form of an interactive network graph and download/export all the results in multiple convenient formats.</p>
            <h3 id="brief-overview-of-the-inputs-and-outputs">Brief overview of the inputs and outputs</h3>
            <h4 id="inputs">Inputs</h4>
            <p>Users need to provide the following inputs:</p>
            <ul>
            <li>A list of human genes.</li>
            <li>A choice of a human genome-scale molecular network: <a href="https://thebiogrid.org/">BioGRID</a>, <a href="https://string-db.org/">STRING</a>, <a href="http://giant.princeton.edu/">GIANT-TN</a>, or <a href="https://string-db.org/">STRING-EXP</a>.</li>
            <li>A choice of how the network connections are represented as features in the supervised machine learning model: Adjacency, Influence, or Embedding.</li>
            <li>A choice of whether the input genes represent a cellular process/pathway or a disease. This choice informs what other genes in the network GenePlexus will use (as the &quot;negative class&quot;) to contrast against the user-provided genes (the &quot;positive class&quot;).</li>
            <li>A unique user supplied job name (optional).</li>
            </ul>
            <h4 id="outputs">Outputs</h4>
            <p>GenePlexus uses the inputs provided by the user to return the following outputs:</p>
            <ul>
            <li>A prediction for every gene in the network on whether it belongs to the positive class, as defined by the user-supplied input gene file.</li>
            <li>The similarity of the custom machine-learning model (trained on the user-supplied input gene file) to other machine-learning models trained using lists of genes annotated to biological process terms in the <a href="http://geneontology.org/">Gene Ontology</a> or diseases in the <a href="https://www.disgenet.org/">DisGeNet</a> database.</li>
            <li>An interactive graph of the network connectivity of the genes with the highest prediction scores</li>
            <li>A summary table of how the user-supplied input gene list is converted to <a href="https://www.ncbi.nlm.nih.gov/Web/Search/entrezfs.html">Entrez</a> gene identifiers and which of those <a href="https://www.ncbi.nlm.nih.gov/Web/Search/entrezfs.html">Entrez</a> genes were in the chosen network.</li>
            </ul>
            <p>For detailed information on the inputs and outputs see the <a href="add link to help page here">Help page</a>.</p>
            <h2 id="overview-of-the-working-and-motivation-of-geneplexus">Overview of the working and motivation of GenePlexus <sup>&beta;eta</sup></h2>
            <p>The goal of GenePlexus is, given a list of genes, predict the association of any gene in the human genome to that list of genes based on the patterns of their connectivity in an underlying genome-scale gene interaction network. This goal is accomplished by casting this as a binary classification machine learning problem, where the user-supplied gene list is considered the &quot;positive class&quot;, and a set of carefully chosen genes are automatically assigned to the &quot;negative class&quot;. GenePlexus then uses a <a href="https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html">regularized logistic regression classifier</a> to train the model that can distinguish genes in the positive class from the negative class.</p>
            <p>The motivation for this network-based approach comes from the difficulty in choosing what features to use for a given machine learning problem. For example, if one wanted to predict if someone was a Democrat or a Republican then maybe choosing features such as state lived in, age, income, etc., might make sense. However, what would be a good feature set to use to predict if a person would like the movie Cinderella? Would the same feature set be able to be used to predict if someone wanted to buy a refrigerator? A powerful method is to forgo traditional feature design and instead use the behaviors/preferences of people within a social network to make predictions. When the problem is cast this way, the features of the machine learning model are always the same; the connections of people to each other in the social network. The only thing that changes with each machine learning problem is the definition of what constitutes the positive and negative classes.</p>
            <p>Designing a feature set for a given problem in genetics is very difficult, but luckily biologists have been studying the interaction of genes and proteins for many years and there exist genome-wide scale molecular networks representing this vast knowledge. GenePlexus uses these networks as the features in its machine learning models, and the user defines the machine learning problem they want to solve by supplying the genes that belong to the positive class and indicating the type of genes that should constitute the negative class. Once these classes are defined, GenePlexus spins up a virtual machine that can handle the GBs worth of data used to train a machine-learning model specific to the user-supplied gene list. Interactive results are then retrievable using the custom job name provided upon running the model.</p>
            <h2 id="previous-published-work-that-has-used-this-approach-of-network-based-gene-classification">Previous published works that have used this approach of network-based gene classification</h2>
            <ul>
            <li>Liu R, Mancuso CA, Yannakopoulos A, Johnson KA, Krishnan A. (2020) <a href="https://academic.oup.com/bioinformatics/article/36/11/3457/5780279">Supervised learning is an accurate method for network-based gene classification</a> <em>Bioinformatics</em> 36:3457-3465.</li>
            <li>Krishnan A, Zhang R, Yao V, Theesfeld CL, Wong AK, Tadych A, Volfovsky N, Packer A, Lash A, Troyanskaya OG. (2016) <a href="https://www.nature.com/articles/nn.4353">Genome-wide prediction and functional characterization of the genetic basis of autism spectrum disorder</a> <em>Nature Neuroscience</em> 19:1454-1462.</li>
            <li>Greene CS, Krishnan A, Wong AK, Ricciotti E, Zelaya RA, Himmelstein DS, Zhang R, Hartmann BM, Zaslavsky E, Sealfon SC, Chasman DI, FitzGerald GA, Dolinski K, Grosser T, Troyanskaya OG. (2015) <a href="https://www.nature.com/articles/ng.3259">Understanding multicellular function and disease with human tissue-specific networks.</a> <em>Nature Genetics</em> 47:569-576.</li>
            </ul>
            <h2 id="license">License</h2>
            <p>The results of the GenePlexus webserver are licensed under the <a href="https://creativecommons.org/licenses/by/4.0/legalcode">Creative Commons License: Attribution 4.0 International</a></p>

            <h3 class="mt-3" id="website">Website Development</h3>

            <p>The GenePlexus backend code and pre-trained models were generated by Dr. Christopher Mancuso and Remy Liu, Department of Computational Mathematics, Science and Engineering, Michigan State University
            </p>
            <p>This GenePlexus website and cloud engineering was done by Douglas Krum, Patrick Bills, and Jacob Newsted, Data Science Group, Enterprise Services, Michigan State University. 
            </p>
        </div>
    </div>
{% endblock %}