{% extends "base-sidenav.html" %}

{% block title %}Home{% endblock %}

{% block head %}

{% endblock %}

{% block app_content %}

<style >
    /* index.html css tweaks.  fas & far are for font awesome */
    .fas { padding-right:1em }
    .far { padding-right:1em }
    h4 { margin-top: 1em;}
    /* the main text on the top of the page */
    .lead-in { font-size:1.4em;font-weight: 300; }

</style>

<div class="ps-3 pe-4">

    <div class="row bg-secondary text-white  rounded p-3">
    
        <div class="col-12">
            <h1 class="text-light" id="geneplexus-help-page" >GenePlexus</h1><!-- class="text-center" -->
            <p class="lead-in">The GenePlexus webserver enables researchers to predict novel genes similar to their genes of interest based on their patterns of connectivity in human genome-scale molecular interaction networks.</p>
        </div>
        <!-- <div class="col-8">
            
        </div>
     -->
    </div>
    
    <div class="row align-content-center">
    <img class="d-block mx-auto col-10"  src="{{ url_for('static',filename='geneplexus_summary_v2.png')}}" >
    </div>
    <h4>Using GenePlexus</h4>
    <ul class="list-group fa-ul" style="margin-left:0;margin-top:1em;">
        <li><i class="fas fa-table"></i>Click ‘Add Genes’ in the panel on the left to start running GenePlexus on your own genes of interest!</li>
        <li><i class="fas fa-video"></i>Watch the <a href="https://youtu.be/ZKmyt7m3QPk">GenePlexus tutorial</a>.</li> <!-- https://academic.oup.com/bioinformatics/article/36/11/3457/5780279 -->
        <li><i class="fas fa-info-circle"></i>See the <a href="{{ url_for('about') }}">About page</a> for an overview of how GenePlexus works.</li>
        <li><i class="far fa-question-circle"></i>See the <a href="{{ url_for('help') }}">Help page</a> for detailed help on the various inputs and outputs.</li>
        <li><i class="fas fa-project-diagram"></i>See an <a href="{{url_for('job', jobname='example')}}">Example run</a> where GenePlexus was used to train a model and predict novel genes associated with the disease (see more below).</li>
        <li><i class="fas fa-pen-square"></i>Write to us at help[at]geneplexus.net with feedback on the usability of the web-server and how it helped your research.</li>
    </ul>
   
    <h4>About GenePlexus</h4>

    <p>When a user provides a set of genes to GenePlexus, it trains a custom machine learning (ML) model that captures the patterns of network connectivity of the user’s genes in contrast to other genes in the network. Based on this ML model Geneplexus will:</p>
    <ol>
        <li>Predict other genes in the network that are similar to the input genes based on their network connectivity.</li>
        <li>Help interpret the custom trained ML model by comparing it to pre-trained models for various biological processes and diseases, and</li>
        <li>Provide a visualization of the top predictions in the form of an interactive network graph.
            All the results can be downloaded/exported in multiple convenient formats.</li>
    </ol>
    <!-- <hr class="mt-4"> -->
    <h4>Job Runtime</h4>
    <p>While many jobs will only take a few minutes to complete, some networks and feature types require training machine learning models that use 10s of GBs of memory, have more that 25,000 features and use thousands of positive and negative labeled genes. Therefore, some jobs may take up to 10 minutes or more and this time can vary due to latency speeds on the cloud server.</p>
    <!-- <hr class="mt-4"> -->
    <h4>Example</h4>
    <p>You can visit
    <a href="{{url_for('job', jobname='example')}}">this page</a>
    to see the results of an example run where GenePlexus was used to train a model and predict novel genes associated with the disease <i>primary ciliary dyskinesia (PCD)</i>.</p>
    <p>This run used the adjacency matrix representation of the human STRING network. Since the input genes correspond to a disease, the negative genes were selected based on genes associated with other diseases (from the DisGeNet database).
        The predicted genes are under the “Gene Predictions” tab. It is notable that the gene DNALI1 (dynein axonemal light intermediate chain 1), which was not included in the original gene list, is strongly predicted to be associated with PCD.</p>
    <p>The “Similarities to…” tabs show that the model trained for PCD is similar to those trained for other diseases such as ciliopathy as well as models trained for biological processes that deal with dynein complex and arm assembly.</p>
    <p>The "Network Graph" tab shows the network connectivity of up to the top 50 genes based on prediction probability score (an example is seen below where blue nodes are previously known associations and orange nodes are novel predicted associations).</p>
    <img class="img-fluid" src="{{ url_for('static',filename='geneplexus_example_screenshot.png')}}">
    <!-- <hr class="mt-4"> -->
    <h4>Underlying Model</h4>
    <p style="margin:0">The network-based approach implemented in GenePlexus has been extensively benchmarked in</p>
    <a href="https://academic.oup.com/bioinformatics/article/36/11/3457/5780279">Supervised learning is an accurate method for network-based gene classification</a>
    <p style="margin:0">R Liu*, CA Mancuso*, A Yannakopoulos, KA Johnson, A Krishnan</p>
    <p style="margin:0">Bioinformatics 36 (11), 3457-3465.</p>


</div> <!-- content wrapper-->

{% block scripts %}
<script src="{{url_for('static', filename='index.js')}}"></script>
{% endblock %}

{% endblock %}